template_feed/Microsoft.DotNet.Common.ProjectTemplates.6.0/content/ConsoleApp-CSharp/.template.config/template.json

```diff
@@ -0,0 +1,92 @@
+{
+    "$schema": "http://json.schemastore.org/template",
+    "author": "Microsoft",
+    "classifications": [ "Common", "Console" ],
+    "name": "Console Application",
+    "generatorVersions": "[1.0.0.0-*)",
+    "description": "A project for creating a command-line application that can run on .NET Core on Windows, Linux and macOS",
```

Comment:
A project for creating a command line application that uses the latest C# features and runs on .NET Core on Windows, Linux and macOS.